CRM/Mailing/Config.php

```diff
@@ -27,6 +27,8 @@ class CRM_Mailing_Config {
   // special value for mail bulk inserts to avoid
   // potential duplication, assuming a smaller number reduces number of queries
   // by some factor, so some tradeoff. CRM-8678
+  // dev/core#1768 Remove this after Dec 2020.
+  // Replaced with civimail_sync_interval.
   const BULK_MAIL_INSERT_COUNT = 10;
```

Comment:
@spalmstr should this be used as the default of the setting?

Reply:
The default value is 10!
```
 // dev/cor#1768 Allow mailer sync interval to be configured by the
  // adminstrator.
  'civimail_sync_interval' => [
    'group_name' => 'Mailing Preferences',
    'group' => 'mailing',
    'name' => 'civimail_sync_interval',
    'type' => 'Integer',
    'quick_form_type' => 'Element',
    'html_type' => 'text',
    'html_attributes' => [
      'size' => 4,
      'maxlength' => 8,
    ],
    'default' => 10,
    'title' => ts('Database Update Frequency'),
    'add' => '5.28',
    'is_domain' => 1,
    'is_contact' => 0,
    'description' => ts('The frequency that CiviMail updates its sent mail database.'),
    'help_text' => 'CiviMail records email sent at the frequency you specify.  If you set it to 1, it will update the database every time it sends an email.  This ensures that emails are not resent if the batch job fails, but this may cause a performance hit, particularly for large jobs.',
  ],
```